pkg/web/transactions.go

```diff
@@ -150,11 +154,12 @@ func (s *Server) TransactionDetail(c *gin.Context) {
 	})
 }
 
-func (s *Server) AcceptTransactionPreview(c *gin.Context) {
+func (s *Server) UpdateTransaction(c *gin.Context) {
```

Comment:
Just doing a bit of re-organization; I didn't rename this -- just moved things around and added banner comments.